pip/compat/__init__.py

```diff
@@ -23,6 +23,26 @@
         ipaddress.ip_network = ipaddress.IPNetwork
 
 
+try:
+    import sysconfig
+
+    def get_stdlib():
+        paths = [
+            sysconfig.get_path("stdlib"),
+            sysconfig.get_path("platstdlib"),
+        ]
+        return set(filter(None, paths))
```

Comment:
Readability, `None` is a special case that doesn't really make sense.

On Thu, Jan 21, 2016 at 8:17 AM, Donald Stufft <notifications@github.com>
wrote:

> In pip/compat/__init__.py
> <https://github.com/pypa/pip/pull/3409#discussion_r50398704>:
>
> > @@ -23,6 +23,26 @@
> >          ipaddress.ip_network = ipaddress.IPNetwork
> >
> >
> > +try:
> > +    import sysconfig
> > +
> > +    def get_stdlib():
> > +        paths = [
> > +            sysconfig.get_path("stdlib"),
> > +            sysconfig.get_path("platstdlib"),
> > +        ]
> > +        return set(filter(None, paths))
>
> Why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pypa/pip/pull/3409/files#r50398704>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084